pkg/bundle/additional_test.go

```diff
@@ -0,0 +1,28 @@
+package bundle
+
+import (
+	"os"
+	"testing"
+)
+
+func Test_GetAdditional(t *testing.T) {
+
+	cfg := &BundleSpec{
+		Mirror: Mirror{
+			BlockedImages:    []string{"alpine"},
+			AdditionalImages: []string{"docker.io/library/alpine:latest", "docker.io/library/busybox:latest"},
```

Comment:
Is there a way to create a fake mirrorer with `oc` libs? Our CI may hit docker limits fairly quickly like this.

Reply:
@estroz Good point. I will into this. Checking some `oc` tests will probably be helpful here.

Reply:
In the meantime we can probably merge these tests as-is, just to get this stuff in.